bin/spiced/src/main.rs

```diff
@@ -63,7 +94,14 @@ fn main() {
 
 async fn start_runtime(args: spiced::Args) -> Result<(), Box<dyn std::error::Error>> {
     spiced::in_tracing_context(|| {
-        tracing::info!("Starting runtime {version}", version = get_version_string());
+        if let Some(allocator_name) = get_allocator_name() {
+            tracing::info!(
+                "Starting runtime {version} (allocator: {allocator_name})",
```

Comment:
I added this log to indicate the allocator being used, only if its not the default one.